kolena/_utils/krequests.py

```diff
@@ -143,21 +144,22 @@ def delete(endpoint_path: str, api_version: int = DEFAULT_API_VERSION, **kwargs:
 def raise_for_status(response: requests.Response) -> None:
     if response.status_code == HTTPStatus.UNAUTHORIZED:
         # HTTP 401 is "unauthorized" but used as "unauthenticated"
-        raise UnauthenticatedError(response.content)
+        raise UnauthenticatedError(response=response)
     if response.status_code == HTTPStatus.NOT_FOUND:
-        raise NotFoundError(response.content)
+        raise NotFoundError(response=response)
     if response.status_code == HTTPStatus.CONFLICT:
-        raise NameConflictError(response.content)
+        raise NameConflictError(response=response)
     if response.status_code == HTTPStatus.UNPROCESSABLE_ENTITY:
         raise IncorrectUsageError(response.content)
 
     try:
         response.raise_for_status()
     except HTTPError:
-        raise RemoteError(f"{response.text} ({response.elapsed.total_seconds():0.5f} seconds elapsed)")
+        error_message = f"{response.text} ({response.elapsed.total_seconds():0.5f} seconds elapsed)"
+        raise RemoteError(response=error_message)  # type: ignore
```

Comment:
Is this right? If the change is for `HttpError`, it is expecting response to be a response object. Maybe 
`RemoteError(error_message, response=response)`?

Reply:
Reverted